homework-submission/main.js

```diff
@@ -38,24 +38,23 @@ class TodoList extends React.Component {
     return (
       <div>
         <h1>static List</h1>
+
         <List description={this.staticList[0].description} deadline={this.staticList[0].deadline} />
+
         <List description={this.staticList[1].description} deadline={this.staticList[1].deadline} />
+
         <List description={this.staticList[2].description} deadline={this.staticList[2].deadline} />
 
         <div>
           <h1>Dynamic List</h1>
           {this.dynamicList.map(todoList => (
-            <div>
-              <li
-                key={todoList.id}
-                style={
-                  todoList.done === true
-                    ? { textDecoration: 'line-through' }
-                    : { textDecoration: '' }
-                }
-              >
-                {todoList.description},{todoList.deadline}
-              </li>
+            <div
+              key={todoList.id}
+              style={
+                todoList.done === true ? { textDecoration: 'line-through' } : { textDecoration: '' }
+              }
+            >
+              {todoList.description},{todoList.deadline}
             </div>
```

Comment:
Thank you for your changes, no point for any else changes, I will not make any further comments,

but starting from line 51 ending here are exactly the same content of the `List` Component, so you could have used it instead, with the style adjustment on that Component:
```javascript
<List description={todoList.description} deadline={todoList.deadline} />
```

Note: you have to pass the style `object/condition/or even the done property` to the Component via props.

**Good Work!**